build.gradle

```diff
@@ -2,6 +2,7 @@ buildscript {
   repositories {
     mavenLocal()
     maven { url ossimMavenProxy }
+    mavenCentral()
```

Comment:
Let's remove this one, if there are components you need from mavenCentral, we should modify our nexus so that they can be satisfied.

Reply:
Rgr.  Added that when i began upgrading things.  Seems to compile fine without.